### PR TITLE
Add initial saltstack support

### PR DESCRIPTION
**Pull request by gtmanfred**
Just starting with bootstrapping servers that are accessible via salt-cloud.

Also include running basic salt execution modules running.

Thanks,
Daniel

### CODE REVIEW

**Comment by dzimine** [comment]
LGTM, someone else will look and merge shortly. 

Would be good to have README.md that states that you install it on salt master and just give the samples already found in the action docstring.

File: packs/salt/actions/bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name: 'bootstrap'
+runner_type: 'run-python'
+description: 'Bootstrap servers with salt.cloud runner'
+enabled: true
+entry_point: 'salt_bootstrap.py'
+parameters:
+    instance_id:
+        description: 'Instance id of the server'
+        type: 'string'
+        required: True
```
Comment (lakshmi-kannan) [inline packs/salt/actions/bootstrap.yaml]: Nit: Consistent case. Let's pick true or True for this pack :)

File: packs/salt/actions/client.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        '''
+        CLI Examples:
+
+            st2 run salt.client matches='web*' module=test.ping
+            st2 run salt.client module=pkg.install \
+                    kwargs='{"pkgs":["git","httpd"]}'
+        '''
+        if args is None:
+            args = []
+        if kwargs is None:
```
Comment (lakshmi-kannan) [inline packs/salt/actions/client.py]: Nit: args=[] kwargs={} might be better in the function def?

**Comment by lakshmi-kannan** [comment]
+1 to README.md. Thanks for the pack.

**Comment by gtmanfred** [comment]
Cool, should be all good :+1:

**Comment by lakshmi-kannan** [comment]
Cool. Merging it! Woohoo!

**Pull request merged**